examples/url-shortener.php

```diff
@@ -101,12 +110,30 @@
   might happen here is the access token itself is
   refreshed if the application has offline access.
  ************************************************/
-if ($client->getAccessToken() && isset($_GET['url'])) {
-  $url = new Google_Service_Urlshortener_Url();
-  $url->longUrl = $_GET['url'];
-  $short = $service->url->insert($url);
-  $_SESSION['access_token'] = $client->getAccessToken();
+if ($client->getAccessToken() && isset($_REQUEST['url'])) {
+  if (validateCsrfToken()) {
+    $url = new Google_Service_Urlshortener_Url();
+    $url->longUrl = $_REQUEST['url'];
+    $short = $service->url->insert($url);
+    $_SESSION['access_token'] = $client->getAccessToken();
+  } else {
+    $invalidCsrf = true;
+    generateCsrfToken();
```

Comment:
We can just follow the model of the [other validation functions](https://github.com/michaelbausor/google-api-php-client/blob/5e085f1041467a5ffacf17663911bddceca51f22/examples/url-shortener.php#L26) if this fails:

```php
if (!validateCsrfToken()) {
  echo invalidCsrfTokenWarning();
  return;
}

// and in base.php
function invalidCsrfTokenWarning()
{
    return 'The CSRF token is invalid, your session probably expired. Please refresh the page ';
}
```